src/series/Pokemon.yml

```diff
@@ -110,71 +106,70 @@ games:
         platforms: []
         release_year: null
         sub-series: Core
-    (III-V) Gen III secondary & remakes:
+    (III) Gen III secondary & remakes:
         genres: []
         platforms: []
         release_year: null
         sub-series: Core
-    (IV) Gen IV secondary:
+        comment: Emerald, FireRed / LeafGreen
+    (IV) Pokémon Platinum:
```

Comment:
`(IV) Gen IV secondary` renamed to `(IV) Pokémon Platinum`

comment added for `(III) Gen III secondary & remakes`